zip-codes/scripts/search.js

```diff
@@ -1,9 +1,45 @@
 'use strict';
 
 (function(module) {
-
+  webDB.execute(
+    'SELECT DISTINCT state FROM zips ' +
+    'ORDER BY state DESC;',
```

Comment:
We did that but then the cities that start with numbers appear :P for easy readability (and quicker selection to WA!) we used DESC :D.